trulens_eval/trulens_eval/database/sqlalchemy.py

```diff
@@ -323,6 +323,16 @@ def insert_record(
             logger.info("{UNICODE_CHECK} added record %s", _rec.record_id)
 
             return _rec.record_id
+    
+
+    def batch_insert_record(self, records: List[mod_record_schema.Record]) -> List[str]:
+        """See [DB.insert_record_batch][trulens_eval.database.base.DB.insert_record_batch]."""
+        with self.session.begin() as session:
+            records_list = [self.orm.Record.parse(r, redact_keys=self.redact_keys) for r in records]
+            session.bulk_save_objects(records_list)
```

Comment:
Looking at `insert_record`, that merges things with any existing records with the same record id. This doesn't do that, so that strikes me as inconsistent. Can we maintain the same approach if possible?

Reply:
I am not sure we really need that. cc: @sfc-gh-pmardziel